demo/hacktoberfest-custom-themes/index.html

```diff
@@ -122,27 +122,10 @@ <h5>Theme Year</h5>
             </div>
             <div id="use-case-panel" class="column bg-gray">
                 <div class="column col-12 text-center">
-                    <h4 id="2020-light-title" class="d-hide">Light Hacktoberfest 2020 Theme</h4>
-                    <h4 id="2020-dark-title" class="d-hide">Dark Hacktoberfest 2020 Theme</h4>
-                    <h4 id="2021-light-title" class="d-hide">Light Hacktoberfest 2021 Theme</h4>
-                    <h4 id="2021-dark-title" class="d-hide">Dark Hacktoberfest 2021 Theme</h4>
-                    <h4 id="2022-light-title" class="hacktoberfest-2022-2023 d-hide">Light Hacktoberfest 2022 Theme</h4>
-                    <h4 id="2022-dark-title" class="hacktoberfest-2022-2023 d-hide">Dark Hacktoberfest 2022 Theme</h4>
-                    <h4 id="2023-light-title" class="hacktoberfest-2022-2023 d-hide">Light Hacktoberfest 2023 Theme</h4>
-                    <h4 id="2023-dark-title" class="hacktoberfest-2022-2023 d-hide">Dark Hacktoberfest 2023 Theme</h4>
-                    <h4 id="default-title" class="d-hide">Default Theme</h4>
+                    <h4 id="hacktoberfest-title"></h4>
```

Comment:
**nitpick:** behaviour changes. The `hacktoberfest-2022-202` class is no more added for 2022 and 2023. 

before | now
----- | -----
![image](https://github.com/process-analytics/bpmn-visualization-examples/assets/27200110/f58c9348-6566-4539-8165-099bf754791c) | ![image](https://github.com/process-analytics/bpmn-visualization-examples/assets/27200110/f587669a-08a1-4d76-956c-8dbdff969d4b)